hooks/command

```diff
@@ -136,6 +136,14 @@ do
   if [[ -n "$pod_name" ]]; then
     break
   else
+    set +e
+    jobstatus=$(kubectl get job "$job_name" -o 'jsonpath={.status.conditions[].type}')
+    set -e
+    if [[ "$jobstatus" == "Failed" ]]; then
+      echo "Warning: job failed to create pod"
```

Comment:
```suggestion
      echo "Warning: exiting because no pod was found and job status was Failed"
```
since the pod could have been created at some point